.github/workflows/analysis-coverage.yml

```
@@ -515,7 +515,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        nextcloud: [ 'stable31', 'stable33' ]
+        nextcloud: [ 'stable33', 'stable34' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)